pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -308,15 +309,15 @@ func (r *ReconcileNetworkAddonsConfig) renderObjects(networkAddonsConfig *opv1al
 		if labels == nil {
 			labels = map[string]string{}
 		}
-		labels[opv1alpha1.SchemeGroupVersion.Group+"/version"] = operatorVersionLabel
+		labels[cnaov1.SchemeGroupVersion.Group+"/version"] = operatorVersionLabel
 		obj.SetLabels(labels)
 	}
 
 	return objs, nil
 }
 
 // Validate and returns the previous configuration spec
-func (r *ReconcileNetworkAddonsConfig) getPreviousConfigSpec(networkAddonsConfig *opv1alpha1.NetworkAddonsConfig) (*opv1alpha1.NetworkAddonsConfigSpec, error) {
+func (r *ReconcileNetworkAddonsConfig) getPreviousConfigSpec(networkAddonsConfig *cnaov1.NetworkAddonsConfig) (*cnaov1.NetworkAddonsConfigSpec, error) {
```

Comment:
If we pass `name string, spec *cnao.NetworkAddonsSpec` then we don't need to reference cnaov1 here

Reply:
since I added `shared.NetworkAddonsConfig` then I guess it's not needed now right? or do you prefer I do the change?